modules/tinymce/src/core/main/ts/content/InsertContentImpl.ts

```diff
@@ -22,6 +22,19 @@ import * as SelectionUtils from '../selection/SelectionUtils';
 import { InsertContentDetails } from './ContentTypes';
 import * as InsertList from './InsertList';
 
+const wrappedElements = [ 'pre' ];
+
+const shouldFlatten = (fragment: AstNode, parentNode: Element) => {
+  const currentNode = fragment.firstChild;
+
+  const isAFlattenableTag = Arr.contains(wrappedElements, currentNode.name);
+  const isPastingInTheSameTag = currentNode.name === parentNode.tagName.toLowerCase();
+  // TINY-7719: last children is the bookmarkHtml attached by the function insertHtmlAtCaret
+  const isCopingOnlyOneTag = fragment.firstChild === fragment.lastChild.prev;
```

Comment:
```suggestion
  const isCopingOnlyOneTag = currentNode === fragment.lastChild.prev;
```
We should be reusing the variable here. Also this might make more sense to be called `firstChild` or something instead of `currentNode`.

Reply:
done